modules/selenium/src/test/java/org/testcontainers/junit/ChromeRecordingWebDriverContainerTest.java

```diff
@@ -56,7 +55,7 @@ public void recordingTestThatShouldBeRecordedAndRetainedInFlvFormatAsDefault() t
                     .withNetwork(NETWORK)
             ) {
                 File[] files = runSimpleExploreInContainer(chrome, "PASSED-.*\\.flv");
-                assertEquals("Recorded file not found", 1, files.length);
+                assertThat(files.length).as("Recorded file not found").isEqualTo(1);
```

Comment:
```suggestion
                assertThat(files.length).as("recorded file count").isEqualTo(1);
```

Other `as()` parts in the file as well.

Reply:
BTW I think we could also use `assertThat(files).hasSize(1)` here - shorter and IIRC will print the content of the array on failure